test.js

```diff
@@ -202,6 +203,13 @@ describe('chalk.level', () => {
 		assert.equal(chalk.level, 1);
 		chalk.level = oldLevel;
 	});
+
+	it('should disable colors if they are not supported', () => {
+		return execa('node', ['fixture'])
+			.then(result => {
+				assert.equal(result.stdout, 'test');
+			});
+	});
```

Comment:
So recently we had to mock `supports-color` in our tests since CIs handled them differently. Does this test guarantee that no matter what the environment supports, it runs without colors?